backend/routes/cset_crud.py

```diff
@@ -295,8 +296,8 @@ def route_json_upload_new_container_with_concepts(d: UploadJsonNewContainerWithC
     return response  # todo: return. should include: assigned codeset_id's
 
 
-# TOOD: do the same try/except validate first false -> true fo all these upload routes
 @router.post("/upload-csv-new-cset-version-with-concepts")
+@return_err_with_trace
```

Comment:
@Sigfried Usage of the `return_err_with_trace` decorator.

I wanted to implement this as a replacement for `router.post()`, to kind of wrap around that 'entirely' so that we would just need to use one decorator for all of our routes. But I'm not sure how easy that would be. Something to do in the future. For now, I think we'll want to (a) use this `return_err_with_trace` decorator like this wherever we want to return any errors explicitly, or we could just (b) use it on all routes for now. I guess probably (b).